import-export-cli/utils/constants.go

```diff
@@ -112,7 +112,6 @@ const defaultAPILoggingListEndpoint = "apis?logging-enabled=false"
 const defaultAPILoggingApisEndpoint = "apis"
 
 const DefaultEnvironmentName = "default"
-const DefaultTenantDomain = "carbon.super"
```

Comment:
Let's add the const back and use it when setting the default value if the tenant-id flag is empty

Reply:
added the constant back.